plugins/modules/packaging/language/gem.py

```diff
@@ -194,10 +200,16 @@ def uninstall(module):
         return
     cmd = get_rubygems_path(module)
     environ = get_rubygems_environ(module)
+    ver = get_rubygems_version(module)
     cmd.append('uninstall')
+    if version.parse(ver) >= version.parse('2.5.2'):
+        cmd.append('--norc')
```

Comment:
Won't this break the playbooks for anyone relying on `.gemrc` files?

Reply:
It can break if somebody relies on the OS state but on the other hand we should not rely on it when it's avoidable and give more control to the configuration tool instead. Being more specific, if somebody needs certain settings for rubygems it should be provided by gem module and not external configuration (such as .gemrc or /etc/gemrc).

Reply:
> It can break if somebody relies on the OS state but on the other hand we should not rely on it when it's avoidable and 
What would be the rationale for "we should not rely on it"?

I can easily imagine a scenario where a devops has this playbook that creates (or templates) a `.gemrc` and then run the `gem` module afterwards, making use of the settings they put in the file.

Shouldn't we at least provide an option for the user, so they can turn off that behaviour?

Reply:
> What would be the rationale for "we should not rely on it"?

As the module behavior cannot be predicted if gemrc is used. E.g. the module docs say no-rdoc is false by default but in reality we cannot be sure. Some options may be failing or not behaving it is supposed to and ansble user won't know why.

> Shouldn't we at least provide an option for the user, so they can turn off that behaviour?

Good point. That would be another feature and this is what people really do than the feature should be implemented unless the module options can cover the use-cases.

Reply:
This indeed sounds like a breaking change, though one that's good. How about the following:
1) Add a new option which enables this. Default value is to disable.
2) In community.general 4, remove the default, and print out a deprecation message that the default will change in community.general 5.0.0 to the new value.
3) In community.general 5.0.0, switch the default.

This allows users to switch to the new behavior ASAP, makes sure that eventually the better default is used, and gives a good upgrade path for existing users of the module.

Reply:
Sounds reasonable. How do we deal with openSUSE and bindir in this case, just ignore?

Reply:
Probably just skip that test when OpenSUSE is used, and change that when the new option is added.

Reply:
Is separate PR needed for no_rc option?

Reply:
Since this PR is also for a new feature, not necessarily. (Also the next release is still a bit more than a week away, so we have some time to get the extended PR reviewed and merged as well.)